dashboard/lib/policies/child_account.rb

```diff
@@ -13,6 +13,15 @@ module ComplianceState
 
     # The student's account has been approved by their parent.
     PERMISSION_GRANTED = SharedConstants::CHILD_ACCOUNT_COMPLIANCE_STATES.PERMISSION_GRANTED
+
+    # def self.locked_out?(student)
+    # def self.request_sent?(student)
+    # def self.permission_granted?(student)
+    SharedConstants::CHILD_ACCOUNT_COMPLIANCE_STATES.to_h.each do |key, value|
```

Comment:
I'm no sure if you wanted these comments to be here.

Reply:
Just wanted to provide more clarity in case someone searches these methods